### PR TITLE
Add readme to EDL

### PR DESCRIPTION
**Pull request by DeanArbel**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

fixes: https://github.com/demisto/etc/issues/21956

## Description
Added README to EDL.

### CODE REVIEW

**Comment by DeanArbel** [comment]
@kirbles19 This is waiting for 7 days, any chance you can prioritize this?

**Comment by kirbles19** [comment]
@DeanArbel Done.

File: Integrations/EDL/README.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+This integration provides External Dynamic List (EDL) as a service for the system indicators (Outbound feed).
+
+
+## Use Cases
+---
+1. Generateg feeds to be used on PAN-OS as External Dynamic Lists.
```
Comment (avidan-H) [inline Integrations/EDL/README.md]:
```suggestion
1. Generate feeds to be used on PAN-OS as External Dynamic Lists.
```

File: Integrations/EDL/README.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+2. Create External Dynamic Lists (EDLs) to track on AutoFocus the IP addresses, URLs and domains used by ransomware, known APT groups, and active malware campaigns.
+3. Create External Dynamic Lists to track the IPs and URLs used by Microsoft Office365, or used as tor exit nodes, or used by CDNs and cloud services.
+
+## Configure EDL on Demisto
+
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
+2. Search for EDL.
```
Comment (avidan-H) [inline Integrations/EDL/README.md]: Isn't the `display` name for this integration something else now? Something with `PAN-OS` in it?

Comment (DeanArbel) [reply Integrations/EDL/README.md]: Yeah, Palo Alto Networks PAN-OS EDL Service. you could change it to that, though EDL should also do

**Pull request merged**